rpcs3/Input/dualsense_pad_handler.cpp

```diff
@@ -6,11 +6,11 @@ LOG_CHANNEL(dualsense_log, "DualSense");
 
 namespace
 {
-	//const auto THREAD_SLEEP = 1ms;
-	//const auto THREAD_SLEEP_INACTIVE = 100ms;
+	const auto THREAD_SLEEP = 1ms;
+	const auto THREAD_SLEEP_INACTIVE = 100ms;
```

Comment:
i think you can safely remove these two lines as they are copy paste from DS4 (which doesn't use them anyway)